include/COLLADA2GLTFWriter.h

```diff
@@ -35,14 +40,21 @@ namespace COLLADA2GLTF {
 		std::map<COLLADAFW::UniqueId, GLTF::Mesh*> _skinnedMeshes;
 		std::map<COLLADAFW::UniqueId, GLTF::Image*> _images;
 		std::map<COLLADAFW::UniqueId, std::tuple<std::vector<float>, std::vector<float>>> _animationData;
+		bool _use_uint_indices = false;
 
 		bool writeNodeToGroup(std::vector<GLTF::Node*>* group, const COLLADAFW::Node* node);
 		bool writeNodesToGroup(std::vector<GLTF::Node*>* group, const COLLADAFW::NodePointerArray& nodes);
 		GLTF::Texture* fromColladaTexture(const COLLADAFW::EffectCommon* effectCommon, COLLADAFW::SamplerID samplerId);
 		GLTF::Texture* fromColladaTexture(const COLLADAFW::EffectCommon* effectCommon, COLLADAFW::Texture texture);
 
+		void interpolateTranslation(float* base, std::vector<float> input, std::vector<float> output, signed_index_t index,
+		                            size_t offset, float time, float* translationOut);
+		float getMeshVertexDataAtIndex(const COLLADAFW::MeshVertexData& data, const index_t index);
+		std::string buildAttributeId(const COLLADAFW::MeshVertexData& data, const index_t index, const size_t count);
+
 	public:
 		Writer(GLTF::Asset* asset, COLLADA2GLTF::Options* options, COLLADA2GLTF::ExtrasHandler* handler);
+		void setUseUintIndices(const bool use_uint_indices);
```

Comment:
This is also never used. Is it just leftover from before the property was added to `GLTF::Options`?